index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="styles.css">
+	<meta charset="UTF-8">
+	<title>Microverse trials</title>
+</head>
+<body>
+<main id="main">
+	<div id="title-container">
+		<h1 id="title"> Lionel Andres Messi Cuccittini </h1> 
+		<h3> The new king after Maradona </h3>

```

Comment:
your code: <h3> The new king after Maradona </h3>

what you should put: <h2> The new king after Maradona </h3>

Reply:
- [ ] Very nice you did great work :+1:  but if you tell the student why you give these required changes then its better understanding for student and he/she will implemented required changes easy :slightly_smiling_face: